modules/15_climate/ar6/declarations.gms

```diff
@@ -0,0 +1,47 @@
+*** |  (C) 2006-2023 Potsdam Institute for Climate Impact Research (PIK)
+*** |  authors, and contributors see CITATION.cff file. This file is part
+*** |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+*** |  AGPL-3.0, you are granted additional permissions described in the
+*** |  REMIND License Exception, version 1.0 (see LICENSE file).
+*** |  Contact: remind@pik-potsdam.de
+*** SOF ./modules/15_climate/ar6/declarations.gms
+
+Scalar
+  s15_forcing_budgetiterationoffset "offset for the calculation of iteratively adjusted budget"
+*** to reach specified forcing level in 2100 in W/m^2 -- best guess is 2010 
+*** value of respective forcing category minus 0.7
+  s15_forcing_budgetiterationoffset_tax "offset for the calculation of iteratively adjusted budget"
+*** to reach specified forcing level in 2100 in W/m^2 -- best guess taken from analysis of ERL paper results
+s15_tempOffset2010 "mean temperature in 2010 from MAGICC"
+;
+
+parameters
+p15_oghgf_pfc    	"exogenous forcings from RCP all in W/m^2: PFCs",
```

Comment:
does "all in W/m^2" apply to the other parameters? Because ideally you would add the unit to all parameters separately, which is helpful if you read them from the gdx and don't have them together like they are here.